From 0ae70548d9e2cbaf184fa76775a2fdfc2f971012 Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Tue, 30 Dec 2014 17:35:19 +0100
Subject: [PATCH] ARM: dts: sun9i Add clock frequency to the CPU nodes

Setting the clock will remove the following error at boot
[    0.061192] /cpus/cpu@0 missing clock-frequency property
[    0.061209] /cpus/cpu@1 missing clock-frequency property
[    0.061223] /cpus/cpu@2 missing clock-frequency property
[    0.061237] /cpus/cpu@3 missing clock-frequency property
[    0.061251] /cpus/cpu@100 missing clock-frequency property
[    0.061266] /cpus/cpu@101 missing clock-frequency property
[    0.061283] /cpus/cpu@102 missing clock-frequency property
[    0.061300] /cpus/cpu@103 missing clock-frequency property

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 arch/arm/boot/dts/sun9i-a80.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm/boot/dts/sun9i-a80.dtsi b/arch/arm/boot/dts/sun9i-a80.dtsi
index 68bcaea..b92e258 100644
--- a/arch/arm/boot/dts/sun9i-a80.dtsi
+++ b/arch/arm/boot/dts/sun9i-a80.dtsi
@@ -69,48 +69,56 @@
 		cpu0: cpu@0 {
 			compatible = "arm,cortex-a7";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x0>;
 		};
 
 		cpu1: cpu@1 {
 			compatible = "arm,cortex-a7";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x1>;
 		};
 
 		cpu2: cpu@2 {
 			compatible = "arm,cortex-a7";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x2>;
 		};
 
 		cpu3: cpu@3 {
 			compatible = "arm,cortex-a7";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x3>;
 		};
 
 		cpu4: cpu@100 {
 			compatible = "arm,cortex-a15";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x100>;
 		};
 
 		cpu5: cpu@101 {
 			compatible = "arm,cortex-a15";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x101>;
 		};
 
 		cpu6: cpu@102 {
 			compatible = "arm,cortex-a15";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x102>;
 		};
 
 		cpu7: cpu@103 {
 			compatible = "arm,cortex-a15";
 			device_type = "cpu";
+			clock-frequency= <1000000000>;
 			reg = <0x103>;
 		};
 	};
-- 
2.1.4

